helix-dap/src/types.rs

```diff
@@ -0,0 +1,672 @@
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+use std::{collections::HashMap, path::PathBuf};
+
+#[derive(Debug, PartialEq, Clone, Deserialize, Serialize)]
+#[serde(rename_all = "kebab-case")]
+pub struct DebugTemplate {
+    pub name: String,
+    pub request: String,
+    pub completion: Option<Vec<String>>,
```

Comment:
I'll change this to an mandatory array of objects describing parameters. This will make user experience very good: They'll get arg names and types completion to work with. Defaults as well.